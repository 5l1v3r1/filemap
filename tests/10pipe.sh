#!/bin/sh

fm -v map -i '/test/*' "wc | sed 's,/.*$,,'"
fm ls /test/*/*/*
