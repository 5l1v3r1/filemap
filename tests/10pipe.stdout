 3  9 18 
 5 15 30 
/test/f1.d/wc/stdout
/test/f1.d/wc/stdout.d/sed=20s=2C=2F.=2A=24=2C=2C
/test/f2.d/wc/stdout
/test/f2.d/wc/stdout.d/sed=20s=2C=2F.=2A=24=2C=2C
